### PR TITLE
Implemented Kabonesa and made move to RFG deactivate cards

### PR DESCRIPTION
**Pull request by presheaf**
Behavior is very similar to Test Run.

### CODE REVIEW

**Comment by presheaf** [comment]
#3025

File: src/clj/game/cards/identities.clj [hunk 1]
```diff
@@ -528,6 +528,25 @@
...
 
+   "Kabonesa Wu: Netspace Thrillseeker"
+   {:abilities [{:label "[:click] Install a non-virus program from your stack, lowering the cost by 1 [Credit]"
+                 :cost [:click 1]
+                 :prompt "Choose a program"
+                 :choices (req (cancellable
+                                (filter #(and (is-type? % "Program")
+                                              (not (has-subtype? % "Virus")))
+                                        (:deck runner))))
+                 :msg (str"install a non-virus program from their stack, lowering the cost by 1 [Credit]")
```
Comment (nealterrell) [inline src/clj/game/cards/identities.clj]: Can you put a space between `str` and `"`? It seems to throw off GitHub's syntax highlighting, even if it works in Clojure.

File: src/clj/game/cards/identities.clj [hunk 2]
```diff
@@ -528,6 +528,25 @@
...
+                 :effect (effect (trigger-event :searched-stack nil)
+                                 (shuffle! :deck)
+                                 (install-cost-bonus [:credit -1])
+                                 (runner-install (assoc-in target [:special :kabonesa] true) {:no-cost false}))
```
Comment (nealterrell) [inline src/clj/game/cards/identities.clj]: You can just drop the `{:no-cost}` argument at the end, it's only important when that value is true.

Comment (presheaf) [reply src/clj/game/cards/identities.clj]: Whoops. Guess which card I copied this from. :)

**Pull request merged**